pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -221,6 +231,10 @@ var log *base.LogObject
 // Set from Makefile
 var Version = "No version specified"
 
+var appStatusDisplayFunc AppStatusDisplayFunc
+var appStatusArgs []string
+var model string
```

Comment:
This can be a local variable in Run once you remove the explicit checks for IPC127 below.

Can't appStatusArgs be a local variable in Run as well?

I don't think you need to define appStatusDisplayFunc here; the compiler can deal with it being defined further down the file.